spyder/app/tour.py

```diff
@@ -1255,6 +1255,76 @@ def any_has_focus(self):
              self.tips.hasFocus() or self.canvas.hasFocus())
         return f
 
+
+class OpenTourDialog(QDialog):
+    """Initial Widget with tour"""
+    sig_launch_tour_button_clicked = Signal()
+    # Signal triggered for the 'Dismiss' button
+    sig_dismiss_button_clicked = Signal()
+
+    def __init__(self, parent, tour_function):
+        super().__init__(parent)
+        self.setFixedHeight(170)
+        self.setFixedWidth(350)
+        self.tour_function = tour_function
+
+        # Image
+        images_layout = QHBoxLayout()
+        if is_dark_interface():
+            icon_filename = 'tour-spyder-logo.png'
+        else:
+            icon_filename = 'tour-spyder-logo.png'
+        image_path = get_image_path(icon_filename)
+        image = QPixmap(image_path)
+        image_label = QLabel()
+        screen = QApplication.primaryScreen()
+        device_image_ratio = screen.devicePixelRatio()
+        if device_image_ratio > 1:
+            image.setDevicePixelRatio(device_image_ratio)
+        else:
+            image_height = image.height() * 0.5
+            image_width = image.width() * 0.5
+            image = image.scaled(image_width, image_height, Qt.KeepAspectRatio,
+                                 Qt.SmoothTransformation)
+        image_label.setPixmap(image)
+
+        images_layout.addStretch()
+        images_layout.addWidget(image_label)
+        images_layout.addStretch()
+
+        # Label
+        tour_label = QLabel(
+            _("Welcome to Spyder! <br><br> Check our interactive tour to "
+              "explore some of Spyder's panes and features. "))

```

Comment:
```suggestion
            _("Welcome to Spyder!<br><br>Check out our interactive tour to "
              "explore some of Spyder's panes and features."))
```

Fix phrasing and trim whitespace